pkg/webservice/handlers.go

```diff
@@ -619,6 +654,57 @@ func getQueueApplications(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getPartitionApplicationsByState(w http.ResponseWriter, r *http.Request) {
+	writeHeaders(w)
+	vars := httprouter.ParamsFromContext(r.Context())
+	if vars == nil {
+		buildJSONErrorResponse(w, MissingParamsName, http.StatusBadRequest)
+		return
+	}
+	partition := vars.ByName(strings.ToLower("partition"))
+	appState := vars.ByName(strings.ToLower("state"))
```

Comment:
Partitions are case-sensitive, so drop that.
For `appState`, the proper approach is:
```
appState := strings.ToLower(vars.ByName("state"))
```